<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>732</width>
    <height>648</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="animated">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="button_chooser">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>530</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>选择文件</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_start">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>530</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>开始</string>
    </property>
   </widget>
   <widget class="QLabel" name="icon">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>40</y>
      <width>161</width>
      <height>160</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../images/icon.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>220</y>
      <width>451</width>
      <height>261</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>快速压缩</string>
     </attribute>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>128</x>
        <y>60</y>
        <width>80</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>源文件大小：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>110</y>
        <width>58</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>目标大小：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_size_source">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>60</y>
        <width>181</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>请先选择视频文件</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBox">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>110</y>
        <width>71</width>
        <height>22</height>
       </rect>
      </property>
      <property name="maximum">
       <number>999999</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_size_unit">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>110</y>
        <width>58</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="quick_mod">
     <attribute name="title">
      <string>参数调整</string>
     </attribute>
     <widget class="QLabel" name="label_size">
      <property name="geometry">
       <rect>
        <x>47</x>
        <y>70</y>
        <width>60</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>分辨率：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_bit">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>110</y>
        <width>58</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>码率：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_fps">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>150</y>
        <width>58</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>帧率：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>30</y>
        <width>58</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>原信息</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>31</y>
        <width>58</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>目标</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_size_2">
      <property name="geometry">
       <rect>
        <x>249</x>
        <y>71</y>
        <width>60</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>分辨率：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_fps_2">
      <property name="geometry">
       <rect>
        <x>262</x>
        <y>151</y>
        <width>58</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>帧率：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_bit_2">
      <property name="geometry">
       <rect>
        <x>262</x>
        <y>111</y>
        <width>58</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>码率：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_shape_source">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>70</y>
        <width>121</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>请先选择视频文件</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_bit_source">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>110</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_fps_source">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>150</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_size_source_2">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>190</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="type_mod">
     <attribute name="title">
      <string>格式转换</string>
     </attribute>
    </widget>
   </widget>
   <zorder>icon</zorder>
   <zorder>tabWidget</zorder>
   <zorder>button_chooser</zorder>
   <zorder>button_start</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
